content/blog/2025-03-11-Alive-Peephole-Optimizations

```diff
@@ -0,0 +1,62 @@
++++
+title = "Alive: Provably Correct Peephole Optimizations"
+[extra]
+[[extra.authors]]
+name = "Annabel Baniak"
+[[extra.authors]]
+name = "Katherine Wu"
+[[extra.authors]]
+name = "Max Fan"
+[[extra.authors]]
+name = "Stephanie Ma"
++++
+
+# Alive: Provably Correct Peephole Optimizations
```

Comment:
You don't need this header; the blog post page already displays your title. (You can preview what this looks like by running Zola yourself.)